### DISCUSSION TITLE
求助，打包后跑不起来

### DISCUSSION THREAD

**Discussion by EbonySpace** [simple-robot/Discussions#45] [Q&A]
在本地IDEA上测试可以正常运行，但是上传到服务器上运行就报错了。
![QQ截图20221218072155](https://user-images.githubusercontent.com/63606841/208269654-51d761f7-bd41-44ee-8532-338cca35074d.png)

**Comment by ForteScarlet** [comment]
bot配置文件读取失败，请检查配置文件格式以及其编码格式是否为utf8等相关内容

**Reply by EbonySpace** [reply]
![QQ截图20221218191812](https://user-images.githubusercontent.com/63606841/208295418-dd08fa8c-c221-4847-b8d5-521afcfe80c0.png)

**Reply by ForteScarlet** [reply]
既然本地可以运行，检查服务端各相关配置是否与本地有所差异并排查它们是否会引发此问题
